js/test/Ice/servantLocator/ServantLocatorI.ts

```diff
@@ -82,7 +82,7 @@ export class ServantLocatorI implements Ice.ServantLocator {
         if (current.operation == "ice_ids") {
             throw new Test.TestIntfUserException();
         } else if (current.operation == "requestFailedException") {
-            throw new Ice.ObjectNotExistException();
+            throw new Ice.ObjectNotExistException(current.id, current.facet, current.operation);
```

Comment:
It's not clear to me why you don't use the default ctor here.

In general, the preferred/recommended way to throw *NotExistException from application code is without arguments.

Reply:
I need to fix the default ctor to not generate a message then, it doesn't work with the refactored code.